docs/docs/config/behaviors.md

```diff
@@ -64,18 +64,19 @@ Definition file: [zmk/app/dts/bindings/behaviors/zmk,behavior-hold-tap.yaml](htt
-| `hold-while-undecided`        | bool     | Triggers the hold behavior immediately on press and releases before a tap                                      | false              |
-| `hold-while-undecided-linger` | bool     | Continues to hold the hold behavior until after the tap is released                                            | false              |
-| `hold-trigger-key-positions`  | array    | If set, pressing the hold-tap and then any key position _not_ in the list triggers a tap.                      |                    |
+| Property                      | Type     | Description                                                                                                   | Default            |
+| ----------------------------- | -------- | ------------------------------------------------------------------------------------------------------------- | ------------------ |
+| `#binding-cells`              | int      | Must be `<2>`                                                                                                 |                    |
+| `bindings`                    | phandles | A list of two behaviors (without parameters): one for hold and one for tap                                    |                    |
+| `flavor`                      | string   | Adjusts how the behavior chooses between hold and tap                                                         | `"hold-preferred"` |
+| `tapping-term-ms`             | int      | How long in milliseconds the key must be held to trigger a hold                                               |                    |
+| `quick-tap-ms`                | int      | Tap twice within this period (in milliseconds) to trigger a tap, even when held                               | -1 (disabled)      |
+| `require-prior-idle-ms`       | int      | Triggers a tap immediately if any non-modifier key was pressed within `require-prior-idle-ms` of the hold-tap | -1 (disabled)      |
+| `retro-tap`                   | bool     | Triggers the tap behavior on release if no other key was pressed during a hold                                | false              |
+| `hold-while-undecided`        | bool     | Triggers the hold behavior immediately on press and releases before a tap                                     | false              |
+| `hold-while-undecided-linger` | bool     | Continues to hold the hold behavior until after the tap is released                                           | false              |
+| `hold-trigger-key-positions`  | array    | If set, pressing the hold-tap and then any key position _not_ in the list triggers a tap                      |                    |
+| `hold-trigger-on-release`     | bool     | If set, delays the evaluation of `hold-trigger-key-positions` until key release                               | false              |
```

Comment:
While this diff looks unclear, I only removed the periods at the end of descriptions (which triggered the table reformat) and added the last item.